apps/nextjs-website/src/_contents/send/apiPath.ts

```diff
@@ -2,5 +2,5 @@ import { Path } from '@/lib/types/path';
 
 export const sendApiPath: Path = {
   name: 'API',
-  path: '/send/api',
+  path: '/send/api/send-api',
```

Comment:
Piattaforma Notifiche: API B2B per le Pubbliche Amministrazioni

```suggestion
  path: '/send/api/api-b2b-per-le-pubbliche-amministrazioni',
```